src/main/java/command/DeleteCheatSheet.java

```diff
@@ -13,10 +12,10 @@ public DeleteCheatSheet(Parser parser) {
     @Override
     public void execute() {
         String name = "";
-        if (parser.getDescriptionMap().containsKey(ArgumentEnum.NAME)) {
-            name = parser.getDescriptionMap().get(ArgumentEnum.NAME);
-        }  else if (parser.getDescriptionMap().containsKey(ArgumentEnum.INDEX)) {
-            int index = Integer.parseInt(parser.getDescriptionMap().get(ArgumentEnum.INDEX));
+        if (parser.getDescriptionMap().containsKey(ArgumentFlagEnum.NAME)) {
+            name = parser.getDescriptionMap().get(ArgumentFlagEnum.NAME);
+        }  else if (parser.getDescriptionMap().containsKey(ArgumentFlagEnum.INDEX)) {
+            int index = Integer.parseInt(parser.getDescriptionMap().get(ArgumentFlagEnum.INDEX));
```

Comment:
Actually for this one, do you want me to add a method to remove cheatsheet by index in addition to the current remove by name?

Reply:
It would be best if you could add that as well